fuse_models/include/fuse_models/unicycle_2d_state_cost_function.h

```diff
@@ -147,20 +145,26 @@ class Unicycle2DStateCostFunction : public ceres::SizedCostFunction<8, 2, 1, 2,
       acc_linear_pred_y,
       jacobians);
 
-    residuals[0] = parameters[5][0] - position_pred_x;
-    residuals[1] = parameters[5][1] - position_pred_y;
-    residuals[2] = parameters[6][0] - yaw_pred;
-    residuals[3] = parameters[7][0] - vel_linear_pred_x;
-    residuals[4] = parameters[7][1] - vel_linear_pred_y;
-    residuals[5] = parameters[8][0] - vel_yaw_pred;
-    residuals[6] = parameters[9][0] - acc_linear_pred_x;
-    residuals[7] = parameters[9][1] - acc_linear_pred_y;
+    const double delta_yaw = parameters[6][0] - parameters[1][0];
```

Comment:
To be consistent with naming
```suggestion
    const double delta_yaw_est = parameters[6][0] - parameters[1][0];
```